azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -1520,8 +1523,12 @@ public JobRunnerEventListener() {
       final Props props = ServiceProvider.SERVICE_PROVIDER.getInstance(Props.class);
       final Map<String, String> metaData = new HashMap<>();
       metaData.put("jobId", node.getId());
+      // Flow specific properties
+      final ExecutableFlow flow = node.getExecutableFlow();
```

Comment:
line #1522 above defines the same variable. Can you remove one?

Reply:
Good catch. Missed during conflict resolution.